spec/models/course_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+require File.dirname(__FILE__) + '/../spec_helper'
+
+RSpec.describe Course, type: :model do
+  before(:all) do
+    @user1 = create(:course)
+  end
+
+  it "it valid with valid attributes" do
```

Comment:
Syntax of Rspec already give you 'it' method, so you should start name of test without 'it'